src/styles/impactful.less

```diff
@@ -1,22 +1,29 @@
 @import './base.less';
 #title {
   color: @white;
-  display: inline-block;
-  .leftPaddedText();
```

Comment:
This may have impacted a few other pages, make sure the CSS doesn't break those pages

Example: https://deploy-preview-61--quirky-brown-9f48ad.netlify.app/competitions

Maybe solution here is to just make some custom CSS just for the home page anyway (we previously tried to generally define some CSS but that's not going to work easily here)